app/models/blacklisted_event.rb

```diff
@@ -14,11 +15,8 @@ def audit_creation
     $audit_log.info("Creating blacklisted event [#{event_name}] for provider [#{provider_model}] with ID [#{ems_id}] by user [#{self.class.current_userid}]")
   end
 
-  def enabled=(value)
-    return if enabled == value
-
-    super
-    $audit_log.info("Blacklisted event [#{event_name}] for provider [#{provider_model}] with ID [#{ems_id}] had enabled changed to #{value} by user [#{self.class.current_userid}]")
+  def log_enabling
+    $audit_log.info("Blacklisted event [#{event_name}] for provider [#{provider_model}] with ID [#{ems_id}] had enabled changed to #{enabled} by user [#{self.class.current_userid}]")
```

Comment:
1. I really like the new solution here
2. Do we need this audit trail?
3. Can you add `new_record?` here? (maybe put a `byebug` in this method and create a new record - see if it goes in here. I worry seeding will produce a slew of these)

Reply:
yeah, I'll check

Reply:
Ok, it has the same exact behavior as before.

Seed with empty db... messages showing it was created, see `audit_creation`, then, became enabled.  I'll add a check to change the behavior to only do it unless `new_record?`. I agree, it was a bug before.

Reply:
pushed change to skip if new_record?